src/test/java/uk/ac/cam/cl/dtg/isaac/api/ITConstants.java

```diff
@@ -50,6 +50,10 @@ public final class ITConstants {
   public static final String TEST_TUTOR_PASSWORD = "test1234";
   public static final long TEST_TUTOR_ID = 12L;
 
+  public static final String TEST_PENDING_TEACHER_EMAIL = "pending-teacher@test.com";
+  public static final String TEST_PENDING_TEACHER_PASSWORD = "test1234";
+  public static final long TEST_PENDING_TEACHER_ID = 13L;
```

Comment:
This does not seem to be used anywhere.

Reply:
Fair enough, I was just following the pre-existing pattern. I'll clean up the other unused id constants while I'm at it?

Reply:
Up to you if you want to remove the rest of the unused ones. I personally don't think that they're useful, as we're not really using them anywhere, so I don't see the point of having those variables there.